controller/backing_image_manager_controller.go

```diff
@@ -590,7 +590,7 @@ func (c *BackingImageManagerController) deleteInvalidBackingImages(bim *longhorn
 			continue
 		}
 		log.Debugf("Start to delete the file for invalid backing image %v", biName)
-		if err := cli.Delete(biName); err != nil && !types.ErrorIsNotFound(err) {
+		if err := cli.Delete(biName, bi.Status.UUID); err != nil && !types.ErrorIsNotFound(err) {
```

Comment:
nit: `cli.Delete(biName, biUUID)`

Reply:
The variable `biUUID` should be invalid here hence we cannot use it.